example/config/postmodeling_config_example.yaml

```diff
@@ -0,0 +1,34 @@
+# Postmodeling Configuration File
```

Comment:
Good to see this moved under this path – now it can be briefly named, simply, `postmodeling.yaml` :smile:

Reply:
To be clear, this is not a reason to hold this work in branch (unmerged) – but this can simply be named `postmodeling.yaml`.

See the discrepancy? :smile: 

```
$ ls -1 example/config/
experiment.yaml
feature.yaml
postmodeling_config.yaml
```